scripts/typescript_checks.py

```diff
@@ -439,6 +439,7 @@
     'core/templates/pages/learner-dashboard-page/learner-dashboard-page.import.ts',
     'core/templates/pages/library-page/search-bar/search-bar.component.spec.ts',
     'core/templates/pages/oppia-root/app.module.ts',
+    'core/templates/pages/oppia-root/app-error-handler.ts',
```

Comment:
How difficult would it be to strictly type this? If it's going to take a while, then I'm fine with this addition since this PR is needed to help address flakiness on develop.

/cc @vojtechjelinek

Reply:
I used a explicit type conversation because nothing else was working. See the comment in app-error-handler.ts. I first tried to strictly type the file but that didn't seem possible. That's why I added the file to this list.